roles/marathon/tasks/main.yml

```diff
@@ -1,104 +1,81 @@
----
-# tasks file for marathon
-- name: upload marathon template service
-  template:
-    src: marathon.conf.j2
-    dest: /etc/init/marathon.conf
-    mode: 0755
-  notify:
-    - restart marathon
-  sudo: yes
-  tags:
-    - marathon
+# Install (docker-py) python package as is a docker module dependency.
+- pip: name=docker-py version=1.1.0
```

Comment:
Probablt dont neede but I like it as It helps to keep the role more self-contained